### PR TITLE
EE-7983: 4.4.0-rc1 throws NoClassDefFoundError XMLResourceIdentifier …

### PR DESCRIPTION
**Pull request by fsgonz**
…deploying a mule app in AIX with IBM JDK8.

### CODE REVIEW

File: api-changes.json [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+          "old": "parameter org.w3c.dom.Document org.mule.runtime.dsl.api.xml.parser.XmlConfigurationDocumentLoader::loadDocument(java.util.function.Supplier<javax.xml.parsers.SAXParserFactory>, org.xml.sax.EntityResolver, java.lang.String, java.io.InputStream, ===com.sun.org.apache.xerces.internal.xni.grammars.XMLGrammarPool===)",
+          "new": "parameter org.w3c.dom.Document org.mule.runtime.dsl.api.xml.parser.XmlConfigurationDocumentLoader::loadDocument(java.util.function.Supplier<javax.xml.parsers.SAXParserFactory>, org.xml.sax.EntityResolver, java.lang.String, java.io.InputStream, ===org.mule.apache.xerces.xni.grammars.XMLGrammarPool===)",
+          "oldType": "com.sun.org.apache.xerces.internal.xni.grammars.XMLGrammarPool",
+          "newType": "org.mule.apache.xerces.xni.grammars.XMLGrammarPool",
+          "package": "org.mule.runtime.dsl.api.xml.parser",
+          "classSimpleName": "XmlConfigurationDocumentLoader",
+          "methodName": "loadDocument",
+          "parameterIndex": "4",
+          "elementKind": "parameter",
+          "justification": "Use of internal JDK classes"
```
Comment (elrodro83) [inline api-changes.json:16]: `Use of internal JDK classes ` -> `Avoid use of internal JDK classes `

File: api-changes.json [hunk 2]
```diff
@@ -1,4 +1,23 @@
 {
+  "1.5.0": {
+    "revapi": {
+      "ignore": [
+        {
+          "code": "java.method.parameterTypeChanged",
```
Comment (elrodro83) [inline api-changes.json:6]: this is to be removed after this is cherry picked to 1.4.x, right?

**Review by elrodro83 — CHANGES_REQUESTED** [review]

File: api-changes.json (hunk 1 above)
Comment (fsgonz) [reply api-changes.json:16]: This will be removed.

File: api-changes.json (hunk 2 above)
Comment (fsgonz) [reply api-changes.json:6]: Yes.

**Review by elrodro83 — APPROVED** [review]

**Pull request closed without merging**